algorithms/edac.py

```diff
@@ -1,10 +1,6 @@
 # Inspired by:
 # 1. paper for SAC-N: https://arxiv.org/abs/2110.01548
 # 2. implementation: https://github.com/snu-mllab/EDAC
-
-# The only difference from the original implementation:
```

Comment:
Maybe we should move this comment to the weights' initialization?

Reply:
Actually, there is no difference from original EDAC now, so this comment is outdated.